core/DataTable/Filter/Sort.php

```diff
@@ -90,7 +93,11 @@ public function filter($table)
         $config->primaryColumnToSort   = $sorter->getPrimaryColumnToSort($table, $this->columnToSort);
         $config->primarySortOrder      = $sorter->getPrimarySortOrder($this->order);
         $config->primarySortFlags      = $sorter->getBestSortFlags($table, $config->primaryColumnToSort);
-        $config->secondaryColumnToSort = $sorter->getSecondaryColumnToSort($row, $config->primaryColumnToSort);
+        if (!empty($this->secondaryColumnSortCallback)) {
+            $config->secondaryColumnToSort = call_user_func($this->secondaryColumnSortCallback, $config->primaryColumnToSort);
```

Comment:
Would it make sense to provide the table as well to the callback function? I'm wondering if it could be useful in some case to make the sort column change based on the contents of the report...

Reply:
added $table as additional param. might be useful in some cases...